### PR TITLE
Kirkstone migration

### PR DESCRIPTION
**Pull request by stlachev**
Kirkstone migration:
-   Implemented kas tool for easy making of dependencies
-   Image build is not supported
-   In this merge, we can use only dry-run upon recipes/packages

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/debian/.devcontainer/base.Dockerfile
```
Comment (mikehaller) [inline .devcontainer/Dockerfile:1]: Should we add license header here as well?

**Review by mikehaller — CHANGES_REQUESTED** [review]
Please do the changes

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (stlachev) [reply .devcontainer/Dockerfile:1]: It is ok now

File: kas/.config-kirkstone.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+local_conf_header:
+  meta-leda: |
+    INHERIT:append = " rm_work"
+    INHERIT:remove = " archiver"
+    INHERIT:remove = " cve-check"
+    INHIBIT_PACKAGE_DEBUG_SPLIT = "1"
+repos:
+  # This entry includes the repository where the config file is located
+  # to the bblayers.conf:
+  meta-custom:
```
[the remark this answers is not in the document]
Comment (stlachev) [reply kas/.config-kirkstone.yaml:23]: Removed also from the other config files

File: meta-leda-components/recipes-sdv/packagegroups/packagegroup-sdv-additions.bb [hunk 3]
```diff
@@ -20,7 +20,7 @@ SDV_EXTERNAL_DEPENDS = "\
     sdv-dapr-binaries \
     sdv-initialize-dapr-first-boot \
     sdv-helm-binaries \
-    sdv-container-databroker \
+    sdv-kuksa-val-databroker \
```
Comment (stlachev) [inline meta-leda-components/recipes-sdv/packagegroups/packagegroup-sdv-additions.bb:23]: Why is kuksa-val, instead of container

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,6 @@
+build/
+meta-openembedded
+meta-rauc
+meta-rauc-community
+meta-virtualization
+poky
```
Comment (stlachev) [inline .gitignore:6]: Please, add a comment, why you exclude this folders (build folder is understandable) in order everybody to know

**Review by stlachev — COMMENTED** [review]
Fixed review comments

File: meta-leda-components/recipes-sdv/packagegroups/packagegroup-sdv-additions.bb (hunk 3 above)
Comment (mikehaller) [reply meta-leda-components/recipes-sdv/packagegroups/packagegroup-sdv-additions.bb:23]: The SDV-Container has been contributed to Eclipse Kuksa.VAL in the meantime, so we need to migrate to that.

File: .gitignore (hunk 4 above)
Comment (mikehaller) [reply .gitignore:6]: Makes sense, thanks

**Review by mikehaller — COMMENTED** [review]
Updated gitignore comments

**Review by mikehaller — APPROVED** [review]
lgtm

**Pull request merged**